src/main/scala/com/codacy/rules/ReportRules.scala

```diff
@@ -79,8 +81,8 @@ class ReportRules(coverageServices: => CoverageServices) extends LogSupport {
       if (config.partial) {
         logger.info(
           """To complete the reporting process, call coverage-reporter with the final flag.
-          | Check https://docs.codacy.com/coverage-reporter/adding-coverage-to-your-repository/#multiple-reports
-          | for more information.""".stripMargin
+            | Check https://docs.codacy.com/coverage-reporter/adding-coverage-to-your-repository/#multiple-reports
```

Comment:
I just noticed this link is outdated (although it includes a redirect to the new one):

```suggestion
            | Check https://docs.codacy.com/coverage-reporter/#multiple-reports
```